server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -133,13 +140,20 @@ public void setUp() throws MailboxException {
         mockedMailSpool = mock(MailSpool.class);
         mockedMailFactory = mock(MailFactory.class);
         mockedAttachmentManager = mock(AttachmentManager.class);
+        mockedMailboxManager = mock(MailboxManager.class);
```

Comment:
Why not use the in memory one? Won't it be easier?

Reply:
I prefere to keep the mock behaviour of this test.